app/controllers/design_methods_controller.rb

```diff
@@ -32,8 +32,25 @@ def edit
 
   def update
     @design_method = DesignMethod.find(params[:id])
+
```

Comment:
Similar to case studies, I think this should be a method on the model, eg @design_method.upload_image(file, request)